app/views/articles/_search.html.erb

```diff
@@ -59,6 +59,16 @@
     }
   }
 
+  function addOptionalBanner(query, displayBanner, jobsUrl) {
```

Comment:
Just curious about this function. Is this for our own internal job postings? Also, I'd remove the `if(query)` from this function and put it down below in the Promise's then since the job of this function is to display a banner, not check if it should before it does.

Reply:
Oh snap I didnt include the original screenshot in this PR like the first one, it will show a little banner pointing to our jobs page like this. 

![Screen Shot 2020-06-03 at 1 57 53 PM](https://user-images.githubusercontent.com/1813380/84072118-4c628380-a994-11ea-9fb5-217adc613c61.png)

I brought it up in Slack bc I would constantly type into Chome "dev.to jobs" when I wanted to get a job link to send to someone and I always ended up on dev.to with jobs as my search query which I didn't want. This ensures that anyone who does that can easily get to our job page.